scheduler/scheduler-server/src/test/resources/functionaltests/descriptors/Task_script_variables.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This test should be extended by creating dependent tasks and verifying variable propagation scenarios.